tests/profiles/helm/values-istio-default.yaml

```diff
@@ -1 +1,2 @@
 # default profile will use all the settings in global.yaml
+certmanager: {}
```

Comment:
I found the `certmanager` and `nodeAgent` are enabled for operator with default and dmeo profiles: https://github.com/istio/operator/blob/master/data/profiles/default.yaml#L185-L190
I'll log a separate issue to track that.

Reply:
looks like you fixed this in https://github.com/istio/operator/pull/153, I will skip it here.